dance/dance_pipeline.py

```diff
@@ -370,6 +372,12 @@ def _sort_molecules_by_fingerprint(input_oeb: pathlib.Path, output_oeb: pathlib.
             Directory in which to create any temporary files.
         """
         # Create a temporary directory for storing sorted chunks of molecules.
+        # NOTE: Windows "equivalent" of Linux's and MacOS's /tmp dir is actually
```

Comment:
Can you add this as a note in the docstring? Use the Sphinx note directive, like here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html?highlight=.. note#directive-note

No one will read the comment right now, but if it is in the docstring, it will show up in the documentation (https://dance.readthedocs.io), so people can see it.

Thanks.